imports/api/questions.js

```diff
@@ -35,7 +35,7 @@ const questionPattern = {
   // null if question template, sessionId if copy attached to question
   sessionId: Match.Maybe(Helpers.MongoID),
   // null if prof created, populated for students creating question for enrolled course
-  courseId: Match.Maybe(Helpers.MongoID),
+  courseId: String,
```

Comment:
keep as maybe, for backwards compatibility. Will something else break if a question does not have a courseId?

Reply:
Nothing breaks if a question does not have a courseId. However, any previous question that was made before the update is invisible due to the question library only showing questions in the current course.